mcd.graphql

```diff
@@ -157,12 +157,12 @@ type FlipEvent {
   tx:      Tx        #                desc: transaction meta
 }
 
-# Pit.file(ilk _).call   -> create FileEvent id: e.ilk
-# Pit.file(what _).call  -> create FileEvent id: e.address
-# Cat.file(ilk _).call   -> create FileEvent id: e.ilk
-# Cat.file(what _).call  -> create FileEvent id: e.address
-# Drip.file(ilk _).call  -> create FileEvent id: e.ilk
-# Drip.file(what _).call -> create FileEvent id: e.address
+# Vat.file(ilk _).call  -> create FileEvent id: e.ilk
+# Vat.file(what _).call -> create FileEvent id: e.address
+# Cat.file(ilk _).call  -> create FileEvent id: e.ilk
+# Cat.file(what _).call -> create FileEvent id: e.address
+# Jug.file(ilk _).call  -> create FileEvent id: e.ilk
+# Jug.file(what _).call -> create FileEvent id: e.address
```

Comment:
It looks like there are `file` functions on the `Vow` - do we need to include those here?

Reply:
yes, i think so, but that should be prioritized with the rest of the work for auctions